can/parser_pyx.pyx

```diff
@@ -184,9 +184,7 @@ cdef class CANDefine():
       defs = def_val[1::2]
 
       if address not in dv:
-        dv[address] = {}
         msgname = address_to_msg_name[address]
-        dv[msgname] = {}
```

Comment:
`dv` is `defaultdict(dict)` and these key are accessed below, so explicit init to `dict` is not needed